### PR TITLE
Improve VSCode debugging documentation

### PR DESCRIPTION
**Pull request by mzjp2**
## Description

The current recommended configuration is inflexible and doesn't allow for specifying CLI arguments. It also requires an `.env` file to be set-up.

I've replaced this with a different approach that is more flexible and intuitive, and doesn't require an `.env` file that fiddles around with the `PYTHONPATH` (though I've left the `.env` file instructions in for safety, but qualified it with "may need to be used").

I've also fixed the Windows settings for the `.env` file - for the same reason that on Unix, you don't need `export A=B` in the `.env` file (you instead need just `A=B`) you don't need `set A=B` on Windows.

## Checklist

- [x] Read the [contributing](https://github.com/quantumblacklabs/kedro/blob/master/CONTRIBUTING.md) guidelines
- [x] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Updated the documentation to reflect the code changes
- [x] Added a description of this change and added my name to the list of supporting contributions in the [`RELEASE.md`](https://github.com/quantumblacklabs/kedro/blob/master/RELEASE.md) file
- [x] Added tests to cover my changes

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

**Review by lorenabalan — APPROVED** [review]
Nice one, thank you! 🚀

**Review by MerelTheisenQB — APPROVED** [review]
Awesome, thanks @mzjp2 🤩

**Comment by lorenabalan** [comment]
Hi @mzjp2 , happy to merge this once the merge conflicts have been fixed. 👌

File: kedro/extras/datasets/geopandas/geojson_dataset.py
Comment (lorenabalan) [inline kedro/extras/datasets/geopandas/geojson_dataset.py:102]: That also works.

```suggestion
                https://geopandas.org/docs/reference/api/geopandas.GeoDataFrame.to_file.html
```

File: RELEASE.md
Comment (lorenabalan) [inline RELEASE.md:29]:
```suggestion
```
We typically have a catch-all for small documentation improvements.
Pre-commit is also only affecting developers, not users, so not needed to be mentioned in release notes.

**Review by lorenabalan — APPROVED** [review]
Minor things, won't stall the merge for this.

**Pull request merged**
